pkg/test/framework/test.go

```diff
@@ -34,7 +34,8 @@ type Test struct {
 	s           *suiteContext
 	requiredEnv environment.Name
 
-	ctx *testContext
+	ctx             *testContext
+	childIsParallel bool
```

Comment:
Some comments here explaining why childIsParallel would be good. (I understood from our discussion, but it is not apparent when looking into the framework code. Somebody can accidentally try to "fix" it.